generate/base.py

```diff
@@ -106,6 +110,7 @@ def main(
     strategy: str = "auto",
     devices: int = 1,
     precision: Optional[str] = None,
+    compile: bool = False,
```

Comment:
We expose this as optional because of the impact during warm up, right?

Reply:
Yes, warmup can take a long time (1-2 min).